scripts/linux.sh

```diff
@@ -12,7 +12,7 @@ get_uuids() {
 }
 
 get_mountpoints() {
-  grep "^$1" /proc/mounts | cut -d ' ' -f 2
+  grep "^$1" /proc/mounts | cut -d ' ' -f 2 | sed 's,\\040, ,g'
```

Comment:
Oh, right – don't mind me then, my brain didn't make the jump back from 32 when I read it as octal :)